domain/src/main/java/tachiyomi/domain/manga/interactor/NetworkToLocalManga.kt

```diff
@@ -14,11 +14,6 @@ class NetworkToLocalManga(
                 val id = insertManga(manga)
                 manga.copy(id = id!!)
             }
-            !localManga.favorite -> {
-                // if the manga isn't a favorite, set its display title from source
-                // if it later becomes a favorite, updated title will go to db
-                localManga.copy(title = manga.title)
```

Comment:
Should just upsert the value here which also resolves #923

Reply:
Oh nice, my intention was really aim to load new cover until I found this line of code. Will implement that.